.pre-commit-config.yaml

```diff
@@ -0,0 +1,29 @@
+# See https://pre-commit.com for more information
+# See https://pre-commit.com/hooks.html for more hooks
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v3.2.0
+    hooks:
+    - id: check-yaml
+    - id: check-added-large-files
+      args: ["--maxkb=250"]
+  - repo: https://github.com/Yelp/detect-secrets
```

Comment:
Consider leveraging [GitHub Secret Scanning](https://docs.github.com/en/code-security/secret-scanning/introduction/about-secret-scanning) unless this action is doing something above and beyond what is done with GitHub.

Reply:
No, nothing special -- someone had added this to pre-commit for QCElemental, so I continued it here. These repositories aren't likely to collect secrets in any case. I've enabled the GH secret scanning at QCManyBody and will do so here and at QCEl if the other primary owner agrees.